### PR TITLE
Gun-related things

### PR DESCRIPTION
**Pull request by Morphan1**
Things for guns

### CODE REVIEW

File: ShadowOperations/ServerGame/ItemSystem/CommonItems/BaseGunItem.cs [hunk 1]
```diff
@@ -61,6 +71,11 @@ public override void AltClick(PlayerEntity player, ItemStack item)
         {
         }
 
+        public override void ReleaseClick(PlayerEntity player, ItemStack item)
+        {
+            player.WaitingForClickRelease = false;
```
Comment (mcmonkey4eva) [inline ShadowOperations/ServerGame/ItemSystem/CommonItems/BaseGunItem.cs]: Maybe do this and any other needed shutdown code in `SwitchFrom()`?

File: ShadowOperations/ServerGame/ItemSystem/CommonItems/PistolGunItem.cs [hunk 2]
```diff
@@ -8,7 +8,7 @@ namespace ShadowOperations.ServerGame.ItemSystem.CommonItems
     public class PistolGunItem: BaseGunItem
     {
         public PistolGunItem()
-            : base("pistol_gun", 0.03f, 10f, 0f, 0f, 10f, 7, "9mm_ammo", 0.5f, 1)
+            : base("pistol_gun", 0.03f, 10f, 0f, 0f, 10f, 7, "9mm_ammo", 0.5f, 1, -1f)
```
Comment (mcmonkey4eva) [inline ShadowOperations/ServerGame/ItemSystem/CommonItems/PistolGunItem.cs]: Why do you have `f` on non-decimal input? 0.o

Comment (Morphan1) [inline ShadowOperations/ServerGame/ItemSystem/CommonItems/PistolGunItem.cs]: *shrugs*
there's a few 10f's there so why not

**Comment by Morphan1** [comment]
@mcmonkey4eva good enough?

**Comment by mcmonkey4eva** [comment]
Awesome. Thanks!

**Pull request merged**